lib/adequate_serializer.rb

```diff
@@ -1,6 +1,7 @@
 require 'adequate_serializer/base'
 require 'adequate_serializer/collection'
 require 'adequate_serializer/helper'
+require 'adequate_serializer/controller'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.